go/vt/vtgate/vindexes/cfc_test.go

```diff
@@ -30,94 +30,117 @@ import (
 	"vitess.io/vitess/go/vt/vterrors"
 )
 
-func assertEqualVtError(t *testing.T, expected, actual error) {
-	// vterrors.Errorf returns a struct containing a stacktrace, which fails
-	// assert.EqualError since the stacktrace would be guaranteed to be different.
-	// so just check the error message
-	if expected == nil {
-		assert.NoError(t, actual)
-	} else {
-		assert.EqualError(t, actual, expected.Error())
+func cfcCreateVindexTestCase(
+	testName string,
+	vindexParams map[string]string,
+	expectErr error,
+	expectWarnings []VindexWarning,
+) createVindexTestCase {
+	return createVindexTestCase{
+		testName: testName,
+
+		vindexType:   "cfc",
+		vindexName:   "cfc",
+		vindexParams: vindexParams,
+
+		expectCost:         1,
+		expectErr:          expectErr,
+		expectIsUnique:     true,
+		expectNeedsVCursor: false,
+		expectString:       "cfc",
+		expectWarnings:     expectWarnings,
 	}
 }
 
-func TestCFCBuildCFC(t *testing.T) {
```

Comment:
Renamed and test cases consolidated into `TestCFCCreateVindex`. All other indexes have a similarly named test case `Test[IndexType]CreateVindex`.